SS14.Jetfish/appsettings.yml

```diff
@@ -26,4 +29,4 @@ User:
     Rivulet: https://static.wikitide.net/rainworldwiki/5/5c/Rivulet_portrait.png
     Spearmaster: https://static.wikitide.net/rainworldwiki/8/8a/Spearmaster_portrait.png
     Saint: https://static.wikitide.net/rainworldwiki/6/6e/Saint_portrait.png
-    Watcher: https://static.wikitide.net/rainworldwiki/1/1a/Nightcat_portrait.png
\ No newline at end of file
+    Watcher: https://static.wikitide.net/rainworldwiki/1/1a/Nightcat_portrait.png
```

Comment:
AHEM THE WATCHER AND NIGHTCAT ARE DIFFERENT ~~SCURRETS~~ SLUGCATS, I WILL NOT STAND FOR THIS ERASURE

Reply:
go blame the wiki smh